osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/ActionMapperService.java

```diff
@@ -278,7 +280,15 @@ public BundleMessagesRequestDto mapAllActions(
     final List<ActionDto> actionValueObjectDtoList = new ArrayList<>();
 
     for (final ActionRequest action : bundleMessageRequest.getBundleList()) {
-      actionValueObjectDtoList.add(this.mapActionWithMapper(smartMeter, action));
+      try {
+        actionValueObjectDtoList.add(this.mapActionWithMapper(smartMeter, action));
```

Comment:
Ik verwacht niet dat dit goed genoeg is. Er moeten evenveel responses zijn als dat er actions zijn. 
Lijkt me dat er wel een error response toegevoegd moet worden